src/pages/components/SitePermissionPage.qml

```diff
@@ -0,0 +1,85 @@
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Controls 1.0
+
+Page {
+    id: page
+
+    property string title
+    property string url
+
+    ListModel {
+        id: permissionTypesModel
+        Component.onCompleted: {
+            PermissionManager.requestUriPermissions(url, function(permissions) {
+                for (var i in permissions) {
+                    var type = permissions[i].type
+                    if (type === "geolocation") {
+                        permissionTypesModel.append({
+                                                        //% "Location"
+                                                        title: qsTrId("sailfish_browser-ti-location"),
+                                                        permission: parseInt(permissions[i].capability),
+                                                        type: type
+                                                    })
+                    } else if (type === "popup") {
+                        permissionTypesModel.append({
+                                                        //% "Pop-up windows"
```

Comment:
```suggestion
                                                        //% "Pop-up"
```
I'd not use term "windows" as these are not windows. Further, "Pop-up" is quite well understood in context of web. Comments?